assets/pages/contributions-landing-new/contributionsLanding.jsx

```diff
@@ -0,0 +1,87 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import Footer from 'components/footer/footer';
+import CirclesIntroduction from 'components/circlesIntroduction/circlesIntroduction';
+import Contribute from 'components/contribute/contribute';
+
+import { init as pageInit } from 'helpers/page/page';
+import { renderPage } from 'helpers/render';
+import { detect } from 'helpers/internationalisation/countryGroup';
+
+import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
+
+import CountrySwitcherHeaderContainer from './containers/countrySwitcherHeaderContainer';
+import ContributionSelectionContainer from './containers/contributionSelectionContainer';
+import ContributionPaymentCtasContainer from './containers/contributionPaymentCtasContainer';
+import PayPalContributionButtonContainer from './containers/payPalContributionButtonContainer';
+
+import { createPageReducerFor } from './contributionsLandingReducer';
+
+
+// ----- Redux Store ----- //
+
+const countryGroupSpecific: {
```

Comment:
Maybe call this countryGroupConfig or countryGroupDetails or even just countryGroups?

Reply:
I would not call it `countryGroups` since there is a global object with that name. Another name could be: `countryGroupConstants`

Reply:
Actually good point, I meant to also move this into a separate file.

Reply:
Good point, this could do with a bit of a refactor.